test/validatePositiveNumber.test.ts

```diff
@@ -0,0 +1,16 @@
+import { validatePositiveNumber } from "../src";
+import { NegativeNumberError } from "../src/Errors";
+
+describe("validatePositiveNumber", () => {
+  it("should raise an error if number is negative", () => {
+    expect(() => validatePositiveNumber(-23)).toThrow(NegativeNumberError);
+  });
+
+  it("should return true if number is bigger than zero", () => {
+    expect(validatePositiveNumber(23)).toBeTruthy();
```

Comment:
por que no to be true?

Reply:
Devuelve true si la validacion es correcta

Reply:
perdon, to be true

Reply:
o sea lo que digo es que esto da bien si devolves no se, "holi"

Reply:
no existe  eso jaja. solo se puede chequear con toBeTruthy

Reply:
si, entiendo. Podria hacer algo como toEqual(true)

Reply:
exacto

Reply:
expect(validatePositiveNumber(23)).toBe(true);